trie/triedb/pathdb/nodebuffer.go

```diff
@@ -217,7 +219,7 @@ func (b *nodebuffer) flush(db ethdb.KeyValueStore, clean *fastcache.Cache, id ui
 	}
 	var (
 		start = time.Now()
-		batch = db.NewBatchWithSize(int(b.size))
+		batch = db.NewBatchWithSize(int(float64(b.size) * DefaultBatchRedundancyRate))
```

Comment:
Can we add some comments to explain why we multiply by this  redundancyRate?

Reply:
yes, added